docs/articles/new-expensify/billing-and-plan-types/The-Free-Plan.md

```diff
@@ -30,16 +30,38 @@ Once you’ve completed your company setup, you should have completed the follow
 - Invited members to the workspace
 - Assigned Expensify Cards
 
-# Inviting Members to the Free Plan:
+# Inviting Members to the Free Plan
 - Navigate to the Settings Menu and click **_Members_** to invite your team. You can invite employees one at a time, or you can invite multiple users by listing out their email addresses separated by a comma
 - To use the Expensify Card, you must invite them to your workspace via your company email address (i.e., admin@companyemail.com and NOT admin@gmail.com).
 
 # Managing the Free Plan
-To access your workspace settings, click your profile icon and then on your workspace name.
+## Workspace Settings
 
-This settings menu allows you to manage your workspace members, issue additional Expensify Cards, and utilize this plan’s various bill pay and payment options.
+To access your workspace settings, click your profile icon and then on your workspace name. This settings menu allows you to manage your workspace members, issue additional Expensify Cards, and utilize this plan’s various bill pay and payment options.
+
+## Paying an Expense Report
+- Once a user creates an expense it will automatically be shared with you in a Processing report.
```

Comment:
NAB: At other places, we have a new line after heading 2. But this line and line 54 and 59 doesn't follow that pattern.